lgsm/config-default/config-lgsm/mumbleserver/_default.cfg

```diff
@@ -9,9 +9,7 @@
 #### Game Server Settings ####
 
 ## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
-fn_parms(){
-parms="-fg -ini ${servercfgfullpath}"
```

Comment:
this will as well not work, the cfg variable is initaled after the parms

Reply:
Yes I see what you mean

Reply:
looks good to me now